staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go

```diff
@@ -154,7 +178,17 @@ func (f *FieldManager) Apply(liveObj, appliedObj runtime.Object, manager string,
 
 	internal.RemoveObjectManagedFields(liveObj)
 
-	if object, managed, err = f.fieldManager.Apply(liveObj, appliedObj, managed, manager, force); err != nil {
+	if userAgent == "kubectl" && f.clientSideApplyUpgrader != nil {
```

Comment:
Why would it be nil? OK I see it can be nil in tests.

Reply:
Removed